File: Selenium/Activities/Activity6.py
from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Firefox() as driver:
    driver.get("https://training-support.net/webelements/dynamic-controls")
    print(f"Title of the page: {driver.title}")

    checkbox_element = driver.find_element(By.ID, "checkbox")

    if(checkbox_element.is_selected()):
        print("Checkbox is selected")
    else:
        print("Checkbox is not selected as expected")

    checkbox_element.click()

    if(checkbox_element.is_selected()):
        print("Checkbox is selected as expected")
    else:
        print("Checkbox is not selected")

File: Selenium/Activities/Activity6.java
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity6 {

	public static void main(String[] args) {
		WebDriver driver = new FirefoxDriver();
		
		driver.get("https://training-support.net/webelements/dynamic-controls");
		System.out.println("Title of the page: " +driver.getTitle());
		
		WebElement checkboxElement = driver.findElement(By.id("checkbox"));
		
		if(checkboxElement.isSelected()) {
			System.out.println("Checkbox is selected");
		}
		else {
			System.out.println("Checkbox is not selected as expected");
		}
		
		checkboxElement.click();
		
		if(checkboxElement.isSelected()) {
			System.out.println("Checkbox is selected as expected");
		}
		else {
			System.out.println("Checkbox is not selected");
		}
		
		driver.quit();
	}

}
